From 8c460397f3d7e43a9349d245a129dd9768753a19 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 25 May 2021 17:45:15 +0300
Subject: [PATCH 1545/1921] dts: marvell: cn9130-crb: Set eth2 to 2500BASE-T
 mode

Eth2 port is actually 2500BASE-T, not 2500BASE-X.

Change-Id: I89b961bece2b892c75fb90407b51412028ed1378
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/52607
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/52708
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index 11059d7dca2f..ae5b31081194 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -114,7 +114,7 @@
 &cp0_eth2 {
 	status = "okay";
 	phy = <&nbaset_phy0>;
-	phy-mode = "2500base-x";
+	phy-mode = "2500base-t";
 	phys = <&cp0_comphy5 0x2>;
 };
 
-- 
2.31.1

